packages/grep-workspace/src/node/grep-workspace.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import * as process from 'process';
+import { ILogger } from '@theia/core/lib/common/logger';
+import { TerminalProcess, TerminalProcessOptions } from '@theia/process/lib/node/terminal-process';
+import { ProcessManager } from '@theia/process/lib/node';
+
+import { rgPath } from 'vscode-ripgrep';
+
+
+export const GrepProcessOptions = Symbol("GrepProcessOptions");
+export interface GrepProcessOptions {
+    command: string,
+    args?: string[],
+    options?: object
+}
```

Comment:
Does these options make sense for ripgrep ?

I don't think it needs command since we know what it is ...
args is OK I guess
does it need options?